manifests/init.pp

```diff
@@ -54,7 +58,7 @@
     }].to_yaml %>')
   $policies_data = parseyaml($policies_yaml)
 
-  package { 'rubygem-foreman_scap_client': } ->
+  package { 'rubygem-foreman_scap_client': ensure => $ensure, } ->
```

Comment:
Not sure we need that as a parameter (if it appears only once). Thoughts @ohadlevy ?